transforms/images/polus-imagej-filter-correlate-plugin/README.md

```diff
@@ -0,0 +1,39 @@
+# ImageJ filter correlate
+
+The plugin applies a correlation operation to an input image with a user 
+specified kernel. A correlation operation is similar to a 
+[convolution](https://en.wikipedia.org/wiki/Convolution) operation except the 
+kernel is flipped by 180 degrees.
```

Comment:
The way this is worded is ambiguous. Which operation flips the kernel?

Also, a few lines down you include the same link for convolution. We should remove one of them.